### ISSUE TITLE
Explore multi-threaded writing

### ISSUE THREAD

**Issue by jdidion** [jdidion/xphyle#15]
Consider main thread manages allocating blocks of space within a file, and worker threads that write into those blocks in parallel (using lseek) and report back to the main thread the size of the block. This would result in inflated file size. The question is whether gzip will read past 'holes' in the file.

**Comment by jdidion** [comment]
Closing as part of a stale issue triage. This issue has been open for 6+ years without activity. If this is still relevant, please feel free to reopen or file a new issue.
